tests/lib/rules/no-children-prop.js

```diff
@@ -15,6 +15,7 @@ var RuleTester = require('eslint').RuleTester;
 var parserOptions = {
   ecmaVersion: 6,
   ecmaFeatures: {
+    experimentalObjectRestSpread: true,
```

Comment:
I thought about doing that, but in looking through all of the tests in the project, there isn't a single other case where different `parserOptions` are used for different test cases.  There are eight other test cases that add `experimentalObjectRestSpread: true` to their top-level `parserOptions`, though.

That said, if you think it's important to establish a new pattern here, I'm happy to make the change.